docs/compose/AppSearchModal/README.md

```diff
@@ -77,13 +83,39 @@ import net.skyscanner.backpack.compose.textfield.BpkClearAction
 
 val destination = remember { mutableStateOf(inputText) }
 BpkAppSearchModal(
+    modifier = modifier,
     title = stringResource(id = R.string.destination),
     inputText = destination.value,
     inputHint = stringResource(id = R.string.text_field_hint),
-    results = BpkAppSearchModalResult.Error(...),
-    closeAccessibilityLabel = stringResource(id = R.string.navigation_close),
+    results = BpkAppSearchModalResult.Error(...
+),
+closeAccessibilityLabel = stringResource(id = R.string.navigation_close),
+onClose = {/* close modal*/ },
+onInputChanged = {/* update input*/ },
+clearAction = BpkClearAction("Clear") { destination.value = "" },
+prefix = prefix,
+)
+```
+
+Example of a AppSearchModal with text prefix in search bar
+
+```Kotlin
+import net.skyscanner.backpack.compose.appsearchmodal.BpkAppSearchModal
+import net.skyscanner.backpack.compose.textfield.BpkClearAction
+
+val destination = remember { mutableStateOf(inputText) }
+BpkAppSearchModal(
+    title = stringResource(id = R.string.destination),
+    inputText = destination.value,
+    inputHint = stringResource(id = R.string.text_field_hint),
+    results = BpkAppSearchModalResult.Content(
+        sections = listOf(BpkSection(...)
```

Comment:
and same section/shortcuts issue here

Reply:
There's a missing bracket I've just spotted 🙃

Reply:
no you can see the closing bracket is still on the next line - both sections + shortcuts are part of `content` but the formatting makes it look like shortcuts is outside of it

Reply:
It's the auto formatting, just because `...` isn't real Kotlin code 😮‍💨 must have fixed the same block 3 times over before realising.